services/order.py

```diff
@@ -0,0 +1,34 @@
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+from services.movie_session import get_movie_session_by_id
+
+
+def create_order(
+        tickets: list[dict],
```

Comment:
The type hint `list[dict]` is correct for Python 3.9 and above. Ensure your environment supports this syntax. If not, consider using `List[Dict]` from the `typing` module for compatibility.